src/interface/interface.ts

```diff
@@ -16,6 +16,8 @@ export interface NodeStatistics {
     successful_tasks: number;
     failed_tasks: number;
     total_tasks: number;
+    last_successful_task_id?: ObjectId;
+    last_successful_task_ts?: string;
```

Comment:
It is a little bit weird for me as currently we had:
Successful Proof Statistics
Latest Processing Time:
182.2644 seconds
Latest Timestamp:
08/04/2025, 09:42:52

They are from proof_timing_stats <TimingStatistics>.

info so why we need add last_successful_task_ts?
Need check the proof_timing_stats to not duplicate them